config/_default/config.toml

```diff
@@ -35,7 +35,18 @@ ordered = false
 startLevel = 2
 
 [outputs]
-home = ["HTML", "RSS", "JSON"]
+  page = ["HTML", "SearchIndex"]
+  home = ["HTML", "RSS", "SearchIndex"]
+  section = ["HTML", "RSS", "SearchIndex"]
+  taxonomyTerm = ["HTML", "RSS", "SearchIndex"]
+  taxonomy = ["HTML", "RSS", "SearchIndex"]
+
+[outputFormats]
+  [outputFormats.SearchIndex]
+    mediaType = "application/json"
+    baseName = "index"
+    isPlainText = true
+    notAlternative = true
```

Comment:
What does that do? I guess I based my config from old example.

Reply:
I'm just referenced Hugo docs
https://gohugo.io/templates/output-formats/#output-format-definitions